### ISSUE TITLE
TLS summary grade letter failing for some sites

### ISSUE THREAD

**Issue by linickx** [mozilla/http-observatory-website#189]
Hello, Looks like the TLS Grade fails for some sites. Instead of a grade you get a purple question mark (?)

Broken Examples:
* [www.google.com](https://observatory.mozilla.org/analyze/www.google.com#tls)
* [www.cloudflare.com](https://observatory.mozilla.org/analyze/www.cloudflare.com#tls)

Working Examples:
* [www.facebook.com](https://observatory.mozilla.org/analyze/www.facebook.com#tls)
* [observatory.mozilla.org](https://observatory.mozilla.org/analyze/observatory.mozilla.org#tls)

_Hyperlinks ^above^ go to scan summary pages, not the website_.

Screenshots attached.
![tls-unknown-grade-www_google_com](https://user-images.githubusercontent.com/649661/53652715-6f56a180-3c41-11e9-8e4f-b7d8e6ee9e4f.png)
![tls-unknown-grade-www_cloudflare_com](https://user-images.githubusercontent.com/649661/53652722-74b3ec00-3c41-11e9-8dcc-185d2fd6f423.png)
![tls-working-observatory_mozilla_org](https://user-images.githubusercontent.com/649661/53652724-75e51900-3c41-11e9-81ee-85c1c221b206.png)
![tls-working-facebook_com](https://user-images.githubusercontent.com/649661/53652726-77164600-3c41-11e9-9ade-13fbbea7a6c7.png)

**Comment by april** [comment]
This is correct, the ? is used to indicate non-compliance. Maybe it should be an `N` so that it is less confusing?

**Comment by linickx** [comment]
Oh. I had indeed interpreted (?) as "_don't know_" as opposed to non-compliance.

Yeah... I think "N" would be clear, or maybe just default to the lowest possible grade (F?) ?
